lib/MusicBrainz/Server/Email.pm

```diff
@@ -35,17 +35,29 @@ has 'c' => (
 
 Readonly our $url_prefix => 'https://' . DBDefs->WEB_SERVER_USED_IN_EMAIL;
 
+sub _encode_header {
+    my $header = @_;
+
+    if ($header =~ /[^\x20-\x7E]/) {
+        return encode("MIME-Q", $header);
```

Comment:
Would it be incorrect, or cause any problems, if we always performed this encoding?

Reply:
Not really, it would just add a few characters `=?UTF-8?Q?…?=`, unnecessarily in most cases.
I also noticed a small side effect: space is replaced with underscore.